_i18n/es/resources/user-guides/restore_from_keys.md

```diff
@@ -1,54 +1,54 @@
 {% assign version = '1.1.0' | split: '.' %}
-{% include disclaimer.html translated="false" version=page.version %}
-### Restoring from keys
+{% include disclaimer.html translated="true" version=page.version %}
+### Restaurar desde claves
 
-Restoring a wallet from private keys is pretty simple. If you have the necessary information, with this guide you can completely restore your wallet. Note: you do NOT have to have your password to restore from keys.
+Restaurar un monedero desde las claves privadas es muy simple. Si cuentas con la información necesaria, con esta guía puedes restaurar tu monedero completamente. Nota: no necesitas tener tu contraseña para restaurar desde claves privadas.
 
-You need to have 3 pieces of data from your wallet, or your .keys file which holds this info and the password to decrypt it. The 3 wallet components that you need are:
+Necesitas tener 3 piezas de información de tu monedero, o tu archivo .keys el cual guarda esta información y la contraseña para des-encriptar. Los 3 componentes de tu monedero que necesitas son:
 
-1. **Address**
-2. **Secret Spendkey**
-3. **Secret Viewkey**
+1. **Dirección**
+2. **Clave secreta de utilización**
```

Comment:
It sounds OK but I think there should be a better word for it, I'll be thinking about it for now.

Reply:
What do we have for those translations? I've been using those two words for them (utilización, visualización) because they're similar in structure, and I can't think about something like "gastación" xD.

Or, we can change the style and use something like _clave secreta de compra_ and _clave secreta de vista_, but I'm not sure about the last one hehe.

Reply:
Here is what is being used in the `monero-wallet-gui`.

[Clave secreta de gasto](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts#L409). 

[Clave pública de vista](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts#L399). The suggestion you are making in this one sounds much better "Clave de visualización".

We could open discussions for this fundamental terms in the [Spanish Teminology](https://github.com/monero-ecosystem/monero-translations/blob/master/spanish-terminology.md). This was suggested by swedish contributor so we won't fill this with comments and have a better communication between us.